templates/frag/operation.hbs

```diff
@@ -150,6 +174,9 @@ open func {{{name}}}Request({{{_params}}}) throws -> URLRequest {
     
 {{/with}}
 {{/if}}
+
+    {{>frag/requestSecurity}}
+
     localVarHeaderParameter.forEach { item in localVarRequest.addValue(item.value!, forHTTPHeaderField: item.name) }
 
     if let localVarPrepareRequestBlock = self.configuration.prepareRequestBlock {
```

Comment:
Maybe we should rename `prepareRequestBlock` to `finaliseRequestBlock`... after all _it_ is the final word on the request, whereas `prepare` makes it sound like it's early.